lib/core/docs/storiesOf.md

```diff
@@ -57,6 +59,27 @@ storiesOf('Button', module).add(
 );
 ```
 
+We can set the parameters and decorators for all stories of a component using the `.addParameters` and `.addDecorator` methods. Note that only one `.addParameters` call is needed, where `.addDecorator` needs to be called once for every decorator you add.
+
+```js
+storiesOf('Button', module)
+  .addParameters({backgrounds: {values: [{name: "red" value: "#f00"}]}})
+  .addDecorator((Story) => <div style={{ margin: '3em' }}><Story/></div>)
+  .addDecorator((Story) => <div style={{ height: '600px' }}><Story/></div>)
+  .add('with text', () => <Button onClick={action('clicked')}>Hello Button</Button>)
+  .add('with some emoji', () => (
+    <Button onClick={action('clicked')}>
+      <span role="img" aria-label="so cool">
+        😀 😎 👍 💯
+      </span>
+    </Button>
+  ));
+```
+
+Finally, global parameters for all stories can be set via the parameters export of your `.storybook/previews.js`, see [this page](https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters) for an example.
+
+Global decorators for all stories can be set via the decorators export of your `.storybook/preview.js`, see [this page](https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators) for an example.
+
```

Comment:
@neilyio i think we could do better and probably go with something like:

 " Parameters and decorators can also be used globally, you can define them in your `.storybook/preview.js`. Take a look [here](https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters) to learn more about global parameters and [here](https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators) for global decorators."

Let me know if you're ok with it